include/slurp.h

```diff
@@ -84,6 +84,11 @@ struct slurp_seat {
 	int32_t anchor_x, anchor_y;
 	struct slurp_box selection;
 	bool has_selection;
+
+	// touch:
```

Comment:
The fields used for both pointer and touch should be groupped into a separate section.

Reply:
Here's the fix 6538ddf97a0822135dffe258ecfdb4112710c028